daisy/workflow.go

```diff
@@ -478,6 +436,11 @@ func (w *Workflow) NewIncludedWorkflow() *Workflow {
 	return iw
 }
 
+// ID is the unique identifyier for this Workflow.
+func (w *Workflow) ID() string {
```

Comment:
What is the purpose of adding a randomized ID to a workflow, which is only ever exposed in logs? Just to make the logs easier to parse?

Reply:
Yes. The id already existed and is used as the suffix in the GCS bucket name. It's now also a suffix in the log name, eg. `projects/[project]/logs/daisy-workflow-[workflow.ID()]`. I wanted to add it to the initial console logs to make it easier to find the GCS bucket and Cloud Log. So now when you kick off a new workflow you see `[Daisy] Running workflow "[workflow name]" (id=abcde)` instead of just the name.